modules/chemical_reactions/src/userobjects/ThermochimicaData.C

```diff
@@ -146,6 +154,58 @@ ThermochimicaDataBase<is_nodal>::ThermochimicaDataBase(const InputParameters & p
     for (const auto i : make_range(_n_potentials))
       _el_pot[i] = &writableVariable("output_element_potentials", i);
   }
+  // buffer size
+  const auto dofid_size = std::max(/* send */ 1, /* receive */ 0);
+  const auto real_size =
+      std::max(/* send */ 2 + _n_elements,
+               /* receive */ _n_phases + _n_species + _element_potentials.size() +
+                   _n_vapor_species + _n_phase_elements);
+
+  // set up shared memory for communication with child process
+  auto shared_mem =
+      static_cast<std::byte *>(mmap(nullptr,
+                                    dofid_size * sizeof(dof_id_type) + real_size * sizeof(Real),
+                                    PROT_READ | PROT_WRITE,
+                                    MAP_ANONYMOUS | MAP_SHARED,
+                                    -1 /* fd */,
+                                    0 /* offset */));
+  if (shared_mem == MAP_FAILED)
+    mooseError(
```

Comment:
All of these errors will be prefixed by the object type and name, so your context is extraneous

Reply:
ah right, we changed that for the MooseServer, right?